### PR TITLE
handle errors

### PR DESCRIPTION
**Pull request by yudetamago**

### CODE REVIEW

File: app/src/App.js [hunk 1]
```diff
@@ -60,16 +60,28 @@ class App extends Component {
...
+            onCompileFailed(response)
+          }
+          break
+
+        case 404:
+          onCompileFailed({status: 'failed', message: `Vyper compiler not found at "${compileURL}".`})
+          break
+
+        default:
+          onCompileFailed({status: 'failed', message: `Unknown error has occured at "${compileURL}".`})
```
Comment (nrryuya) [inline app/src/App.js]: How about showing the content of response?

Comment (yudetamago) [reply app/src/App.js]: nice suggestion!

File: app/src/App.js [hunk 2]
```diff
@@ -60,16 +60,29 @@ class App extends Component {
...
+          break
+
+        default:
+          onCompileFailed({status: 'failed', message: `Unknown error has occured at "${compileURL}". ${event.target.responseText}`})
+          break
       }
     })
     request.addEventListener("error", () => {
-      console.error("Network Error")
+      onCompileFailed({status: 'failed', message: `Network error has occured at "${compileURL}".`})
```
Comment (nrryuya) [inline app/src/App.js]: `occurred` ?

Comment (yudetamago) [reply app/src/App.js]: yes

**Pull request merged**